resources/monitoring/templates/kyma-additions/aks-kubelet-monitoring-patch.yaml

```diff
@@ -25,20 +25,30 @@ data:
     metadata:
       labels:
         app: {{ template "prometheus-operator.name" . }}-kubelet
-        {{- include "prometheus-operator.labels" . | indent 8 }}
+    {{- include "prometheus-operator.labels" . | indent 8 }}
```

Comment:
i think this should be indented 2 spaces.

Reply:
fixed